### PR TITLE
FEM: fix .cpp and .h header text

### PR DESCRIPTION
**Pull request by luzpaz**
This is a smaller PR of the recently closed #45
This addresses just `.cpp` and `.h` files

### CODE REVIEW

**Comment by luzpaz** [comment]
@berndhahnebach this is a quick merge because the fixed are obvious

**Comment by berndhahnebach** [comment]
if we gone change headers we should make them all look the same ...

` ` ` 
 /**************************************************************************
 *   Copyright (c) 2015 FreeCAD Developers                                 *
 *   Author: Przemo Firszt <przemo@firszt.eu>                              *
 *   Author: Bernd Hahnebach <bernd@bimstatik.ch>                    *

`

May be even better without Author ... 

What do you think?

**Comment by berndhahnebach** [comment]
damn code formating ...

File: src/Mod/Fem/App/FemSolverObject.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /***************************************************************************
- *   Copyright (c) 2013 J¨¹rgen Riegel (FreeCAD@juergen-riegel.net)        *
+ *   Copyright (c) 2013 Jürgen Riegel (FreeCAD@juergen-riegel.net)         *
```
Comment (berndhahnebach) [inline src/Mod/Fem/App/FemSolverObject.cpp]: AFAIK code files should not have ü, but I'm not 100 % sure

**Comment by berndhahnebach** [comment]
how about this ?

https://github.com/berndhahnebach/FreeCAD_bhb/commit/35ef77c

File: src/Mod/Fem/App/FemSolverObject.cpp (hunk 1 above)
Comment (luzpaz) [reply src/Mod/Fem/App/FemSolverObject.cpp]: `grep -iR riegel` shows that `Jürgen` is correct.

**Comment by luzpaz** [comment]
Answer your other question in https://github.com/berndhahnebach/FreeCAD_bhb/commit/35ef77c#commitcomment-31025235

**Comment by berndhahnebach** [comment]
Than we should go for this in all headers ... 
https://github.com/FreeCAD/FreeCAD/blob/c78ebdec42529f44cf5e104ada6ac316ec1dfbdf/src/Tools/_TEMPLATE_/App/App_TEMPLATE_.cpp#L2

I would not cut the mail on the second line.

**Comment by luzpaz** [comment]
> I would not cut the mail on the second line.

What do you propose for devs with long names?  :nerd_face: 
like
```
/***************************************************************************
 *   Copyright (c) 2013 Jan Rheinländer <jrheinlaender[at]users.sourceforge.net> *  
 *                                                                         *
```

**Comment by berndhahnebach** [comment]
ok convinced, we go for the two lines. 

BTW: how did you get the nice code formating managed?

**Comment by luzpaz** [comment]
> BTW: how did you get the nice code formating managed?

the old-fashioned way, manually :face_with_head_bandage: 

So I guess we're green light to accept this PR ?

**Comment by berndhahnebach** [comment]
now I mean in the code in the github comment. My code looked terrible on github comment.

**Comment by berndhahnebach** [comment]
I would change the (e-mail-adress) to  <e-mail-adress> to really make them look the same. Just like in the template.

**Comment by berndhahnebach** [comment]
github eat my <e-mail-adress>

**Comment by berndhahnebach** [comment]
` would change the (e-mail-adress) to to really make them look the same. Just like in the template.
`

**Comment by berndhahnebach** [comment]
damn my greater and smaller gets eaten all the time ...

**Comment by berndhahnebach** [comment]
I would change the (e-mail-adress) to 

smallercharacter e-mail-adress greatercharacter  

to really make them look the same. Just like in the template.

**Comment by luzpaz** [comment]
Oh! 
yea, you need to learn about github markdown: 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

**Comment by luzpaz** [comment]
> I would change the (e-mail-adress) to  
> smallercharacter e-mail-adress greatercharacter  
> to really make them look the same. Just like in the template.

Yea, I'm going to need an example of what you're aiming for

**Comment by berndhahnebach** [comment]
https://github.com/berndhahnebach/FreeCAD_bhb/commit/1ca6888655cae5ecae3399df49591d90637bf247

**Comment by berndhahnebach** [comment]
accroding https://github.com/FreeCAD/FreeCAD/blob/c78ebdec42529f44cf5e104ada6ac316ec1dfbdf/src/Tools/_TEMPLATE_/App/App_TEMPLATE_.cpp#L2

**Comment by luzpaz** [comment]
ah..ok!
checkout 8d9f9b9

**Comment by berndhahnebach** [comment]
https://github.com/berndhahnebach/FreeCAD_bhb/commit/3e253adad78583d729c1a583b726b4a50630befd

**Comment by berndhahnebach** [comment]
merged

**Comment by luzpaz** [comment]
@berndhahnebach can you rebase `femdev` again ?

**Comment by berndhahnebach** [comment]
https://github.com/FreeCAD/FreeCAD/pull/1744

**Comment by berndhahnebach** [comment]
> @berndhahnebach can you rebase `femdev` again ?

done

**Pull request closed without merging**